iot-hub/Samples/service/ServiceClientSample/ServiceClientSample.cs

```diff
@@ -50,13 +54,50 @@ public async Task RunSampleAsync(TimeSpan runningTime)
 
         }
 
+        private async Task ReceiveMessageFeedbacksAsync(CancellationToken token)
+        {
+            _logger.LogInformation("Starting to listen to feedback messages");
+
+            while (!token.IsCancellationRequested)
+            {
+                var feedbackReceiver = _serviceClient.GetFeedbackReceiver();

```

Comment:
This could be initialized outside the while loop, right?

Reply:
if it's initialized outside, then upon re-initialization, we have to re-initialize it also right? I can do that.